README.md

```diff
@@ -175,3 +199,47 @@ Follow these steps for creating the end of sprint PRs for the court.
 9. After the court approves and merges PR, merge master into develop
 10. Create a release in GitHub as sprint_00x against master and put the same description planned to be in the PR description for the court
 
+# Setup for CircleCI CI/CD Pipeline
+
```

Comment:
Would you please modify the Snyk URLs in the README to replace `/flexion/` with `/ustaxcourt/`, so that the badges reflect the state of the Court's environment, rather than Flexion's?

(If it’s useful to you to have the badges indicate the state of things within Flexion's environment, let me know and we'll find a solution that works for everybody.)

Reply:
Updated!